src/lib.rs

```diff
@@ -0,0 +1,49 @@
+use crate::actors::{delete_actor, get_actors, new_actor};
+use crate::characters::get_characters;
+use actix_web::dev::Server;
+use actix_web::{web, App, HttpServer};
+use dotenvy::dotenv;
+use sqlx::postgres::PgPoolOptions;
+use sqlx::PgPool;
+
+mod actors;
+mod characters;
+
+pub struct AppState {
+    db: PgPool,
+}
+
+pub async fn run() -> Result<Server, std::io::Error> {
```

Comment:
Still planning on renaming this to something like getServer() and/or configuring test database setup as part of this PR?